Lib/fontTools/varLib/__init__.py

```diff
@@ -164,10 +143,12 @@ def _add_avar(font, axes):
 
 		curve.update(zip(keys, vals))
 
-		assert 0.0 in curve and curve[0.0] == 0.0
-		assert -1.0 not in curve or curve[-1.0] == -1.0
-		assert +1.0 not in curve or curve[+1.0] == +1.0
```

Comment:
these should stay as assertions. given the assumptions and the way the normalizing code works, these will always be true. You can't hit them, hence assertion is the correct error